app/components/Form/ProjectContactForm.tsx

```diff
@@ -177,17 +181,6 @@ const ProjectContactForm: React.FC<Props> = (props) => {
           },
         },
       },
-      optimisticResponse: {
-        updateFormChange: {
-          formChange: {
-            ...formChange,
-            newFormData,
-            changeStatus: formChange.changeStatus,
-            // explicitely ignore the field below
-            projectRevisionByProjectRevisionId: undefined,
-          },
-        },
-      },
```

Comment:
Remind me why this needs to be removed?

Reply:
I think because it wasn't doing anything (I don't remember why it wasn't working, though)